src/blueapi/service/facade.py

```diff
@@ -0,0 +1,85 @@
+from abc import ABC, abstractmethod
+from typing import List, Optional
+
+from blueapi.service.model import DeviceModel, PlanModel, WorkerTask
+from blueapi.worker.event import WorkerState
+from blueapi.worker.task import RunPlan
+from blueapi.worker.worker import TrackableTask
+
+
+class BlueskyHandler(ABC):
+    """Interface between web application and underlying Bluesky context and worker"""
+
+    @property
```

Comment:
There is a mix of properties and getters now, is this cleaner overall or more confusing?

Reply:
I don't think it's too bad